app/mailers/internal_mailer.rb

```diff
@@ -6,4 +6,12 @@ def request_created_email
     @edit_url = HOST_AND_SCHEME + '/providers/' + params[:linked_token].token + '/edit'
     mail(template_path: 'provider_mailer', to: INTERNAL_EMAIL, from: @provider.email, reply_to: @provider.email, subject: 'Your help request has been created.')
   end
+
+  def volunteer_response_email
+    @provider = params[:provider]
+    @volunteer = params[:volunteer]
+    @response = params[:response]
+    mail(template_path: "provider_mailer", to: INTERNAL_EMAIL, from: @volunteer.email, reply_to: @volunteer.email,
```

Comment:
I just learned that we can only send emails from known senders that we setup in the Postmark UI. So we'll have to drop the from here. More details here: https://postmarkapp.com/developer/user-guide/managing-your-account/managing-sender-signatures